src/Components/Facility/models.tsx

```diff
@@ -229,3 +226,430 @@ export type ICD11DiagnosisModel = {
+  expected_oxygen_requirement: number;
+  type_b_cylinders: number;
+  type_c_cylinders: number;
+  type_d_cylinders: number;
+  expected_type_b_cylinders: number;
+  expected_type_c_cylinders: number;
+  expected_type_d_cylinders: number;
+};
+
+export type IListDoctorResponse = PaginatedResponse<{
+  id: string;
+  area_text: number;
+  area: number;
+  count: number;
+}>;
+
+export type IUserListFacilityResponse = {
+  id: number;
+  username: string;
+  first_name: string;
+  last_name: string;
+  email: string;
+  user_type: number;
+  doctor_qualification: string;
+  doctor_experience_commenced_on: string;
+  doctor_medical_council_registration: string;
+  created_by: string;
+  home_facility: string;
+  weekly_working_hours: number;
+  local_body: number;
+  district: number;
+  state: number;
+  phone_number: string;
+  alt_phone_number: string;
+  gender: number;
+  age: number;
+  is_superuser: boolean;
+  verified: boolean;
+  home_facility_object: HomeFacilityModel;
+  local_body_object: { id: number } & LocalBodyModel;
+  district_object: DistrictModel;
+  state_object: StateModel;
+  pf_endpoint: string;
+  pf_p256dh: string;
+  pf_auth: string;
+};
+
+export type DeleteModel = {
+  detail?: string;
+};
+
+export type IUserFacilityRequest = {
+  username?: string;
+  first_name?: string;
+  last_name?: string;
+  email?: string;
+  doctor_qualification?: string;
+  doctor_experience_commenced_on?: string;
+  doctor_medical_council_registration?: string;
+  home_facility?: string;
+  weekly_working_hours?: number;
+  phone_number?: string;
+  alt_phone_number?: string;
+  gender?: number;
+  age?: number;
+};
+
+export type IUserFacilityResponse = IUserListFacilityResponse;
+
+export type IFacilityUserResponse = {
+  count: number;
+  next: string | null;
+  previous: string | null;
+  results: {
+    id: number;
+    first_name: string;
+    username: string;
+    email: string;
+    last_name: string;
+    alt_phone_number: string;
+    user_type: number;
+    last_login: string;
+    home_facility_object: HomeFacilityModel;
+    doctor_qualification: string;
+    doctor_experience_commenced_on: string;
+    doctor_medical_council_registration: string;
+    skills: {
+      id: string;
+      name: string;
+      description: string;
+    }[];
+  }[];
+};
+
+export type ILocalBodyResponse = StateModel & {
+  body_type: number;
+  localbody_code: string;
+  district: number;
+};
+
+export interface InventoryItemObjectModel {
+  id: number;
+  default_unit: UnitModel;
+  allowed_units: UnitModel[];
+  tags: UnitModel[];
+  name: string;
+  description: string;
+  min_quantity: number;
+}
+
+export type IInventorySummaryResponse = PaginatedResponse<{
+  id: string;
+  item_object: InventoryItemObjectModel;
+  unit_object: UnitModel;
+  created_date: string;
+  quantity: number;
+  is_low: boolean;
+  item: number;
+}>;
+
+export type IInventoryLogResponse = Omit<
+  IInventorySummaryResponse,
+  "results"
+> & {
+  results: {
```

Comment:
You are omitting results and adding results of different type. Define a new type instead since this approach will simply make the code hard to read and hard to maintain